Makefile

```diff
@@ -19,4 +20,9 @@ validate-meta:
 normalize:
 	for file in $(SOURCES) ; do \
 		ontotools file normalize $$file ; \
-	done
\ No newline at end of file
+	done
+
+fetch:
+	for file in $(SHAPES) ; do \
+    	python fetch.py $$file ; \
+    done
```

Comment:
For-loop formatting. Please fix the indenting. See https://www.cyberciti.biz/faq/bash-for-loop/.